static/js/ducks/observations.js

```diff
@@ -81,6 +81,7 @@ messageHandler.add((actionType, payload, dispatch) => {
 
 export function fetchGcnEventObservations(dateobs, filterParams = {}) {
   filterParams.localizationDateobs = dateobs;
+  filterParams.numPerPage = 1000;
```

Comment:
Is there a reason for that particular number, or is the goal just to get a lot / all of them and 1000 should be enough ?

Reply:
yes exactly. At some point we should maybe rethink a bit the map display, where we could consider aggregating statistics across field to then display in tooltips.